auth/src/main/java/com/firebase/ui/auth/ui/credentials/CredentialSaveActivity.java

```diff
@@ -55,7 +55,7 @@ public void onChanged(@Nullable Resource<Void> resource) {
                     return;
                 }
 
-                onSaveOperation(resource);
```

Comment:
1) Why is `ResourceObserver` not good for this situation?
2) Having a usable `Resource<Void>` is kind of strange ... but maybe I should just get over it,

Reply:
That was you! 😆 Nah, I think we weren't doing it because we didn't want to show a loading dialog, but now that I'm looking at the code, we do have a task running in the background so we should be showing one. 🤷‍♂️, I'll use the `ResourceObserver` 😄.

Reply:
Huh, the void thing is kinda ugly: it messed with our nullability annotations. Not sure how to fix it though without getting rid of the `Resource` for this altogether.